Support sass-embedded as alternative implementation (#124)
jekyll/jekyll-sass-converter: README.md

information about usage, visit the [Jekyll Assets Documentation
page](https://jekyllrb.com/docs/assets/).

### Sass Implementations

#### SassC

By default, Jekyll Sass Converter uses [sassc](https://rubygems.org/gems/sassc)
for Sass implmentation. `sassc` is based on LibSass, and
[LibSass is deprecated](https://sass-lang.com/blog/libsass-is-deprecated).

#### Sass Embedded

[sass-embedded](https://rubygems.org/gems/sass-embedded) is a host for the
[Sass embedded protocol](https://github.com/sass/embedded-protocol). The host

> runs [Dart Sass compiler](https://github.com/sass/dart-sass-embedded) as subprocess

Reviewer: ```suggestion
runs [Dart Sass compiler](https://github.com/sass/dart-sass-embedded) as a subprocess
```

> and communicates with it via message-passing.

Reviewer: ```suggestion
and communicates with the dart-sass compiler via message-passing.
```

but what is _message-passing_?

Reviewer: @ntkme Could you elaborate what you mean by _message-passing_..?

Author: I copied that term from: https://sass-lang.com/blog/libsass-is-deprecated#what-about-portability-and-performance

Specifically, it means sending/receiving protobuf messages via stdio.

> and communicates with the dart-sass compiler via message-passing.

Reviewer: ```suggestion
and communicates with the dart-sass compiler by sending / receiving [protobuf](https://github.com/protocolbuffers/protobuf) messages via stdio..
```

`sass-embedded` is currently experimental and unstable. It requires Ruby 2.6 or higher.

To use the `sass-embedded` implementation, you need to add a dependency on the

> `sass-embedded` gem. For example, `bundle add sass-embedded`. Then, you'll be able to

Reviewer: ```suggestion
`sass-embedded` gem. For example, if you're using a Gemfile, run `bundle add sass-embedded`. Then, you'll be able to
```
